src/hooks/useLoginForm/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  emailValidator,
+  passwordValidator
+} from 'validators';
+
+const useLoginForm = () => {
+  const [inputs, setInputs] = React.useState({
+    email: '',
+    password: ''
+  });
+  const handleSubmit = async (event) => {
+    if (event) {
+      event.preventDefault();
+      if (!emailValidator(inputs.email)) {
+        console.log('incorrect email entered')
+      } else {
+        const {email, password} = inputs
+          const result = await fetch(`${process.env.REACT_APP_AUTH_ENDPOINT}/login`, {
+              method: 'POST',
+              headers: { 'Accept': 'application/json', 'Content-Type': 'application/json' },
```

Comment:
If we're using these same headers in multiple places, it might make sense to create a constant to be imported for us to use for these requests rather than typing them out over and over again

Reply:
Added headers constant